deploy/README.md

```diff
@@ -36,15 +31,15 @@ helm install elastic-operator elastic/eck-operator -n my-namespace --create-name
 View the available settings for customizing the installation.
 
 ```sh
-helm show values elastic/eck-operator
+helm show values ./eck-operator
```

Comment:
This should go above the install step, as the values for installation should be reviewed prior to installation.